extensions/amp-consent/amp-consent.md

```diff
@@ -367,6 +379,8 @@ window.parent.postMessage(
   {
     type: 'consent-response',
     action: 'accept/reject/dismiss',
+    info: /string/ /* optional */,
```

Comment:
This is the `<amp-iframe>` case. No string or metadata will be supported unfortunately.

Reply:
Oops, got confused. Thanks